src/Model/CompositeTypeGo.cs

```diff
@@ -261,6 +261,15 @@ public virtual string Fields()
                 {
                     indented.Append($"{property.Name} - {property.Documentation}".ToCommentBlock());
                 }
+                if (property.Deprecated)
+                {
+                    var message = "This property has been deprecated.";
+                    if (!string.IsNullOrWhiteSpace(property.DeprecationMessage))
+                    {
+                        message = property.DeprecationMessage;
+                    }
+                    indented.Append($"// Deprecated: {message}");
+                }

```

Comment:
should we add a new line to make it a paragraph per the doc recommendation.  [https://blog.golang.org/godoc-documenting-go-code](https://blog.golang.org/godoc-documenting-go-code)

Reply:
Oh good catch we do need a new-line here.  The other cases use WrapComment which will do the right thing.